# Message package, first object defines any aruco detection, the second if those of the ull were detected

bool aruco_detection
bool bull_detection